qwandaq/src/main/java/life/genny/qwandaq/utils/QwandaUtils.java

```diff
@@ -496,36 +496,43 @@ public static boolean mandatoryFieldsAreAnswered(Map<String, Ask> map, BaseEntit
 			String value = baseEntity.getValueAsString(attributeCode);
 
 			// if any are blank, mandatory and non-readonly, then task is not complete
-			if ((mandatory && !readonly))
+			Boolean answered = false;
+			if ((mandatory && !readonly)) {
 				answered = acceptableAnswer(value);
 
-			if(!answered) {
-				break;
+				// not complete if any mandatories are not answered
+				if (!answered) {
+					complete = false;
```

Comment:
Do we need to keep checking which answers aren't complete if we have reached an incomplete field?

Reply:
We will come back to this.